lib/Runtime/Base/FunctionBody.h

```diff
@@ -1159,10 +1195,49 @@ namespace Js
     //
     class FunctionProxy : public FunctionInfo
     {
+    public:
+        typedef RecyclerWeakReference<DynamicType> FunctionTypeWeakRef;
+        typedef JsUtil::List<FunctionTypeWeakRef*, Recycler, false, WeakRefFreeListedRemovePolicy> FunctionTypeWeakRefList;
+
     protected:
         FunctionProxy(JavascriptMethod entryPoint, Attributes attributes, int nestedCount, int derivedSize,
             LocalFunctionId functionId, ScriptContext* scriptContext, Utf8SourceInfo* utf8SourceInfo, uint functionNumber);
         DEFINE_VTABLE_CTOR_NO_REGISTER(FunctionProxy, FunctionInfo);
+
+        enum AUXPTRS : uint8 {
```

Comment:
Also I think the name should be more descriptive to indicate what it is used for. Maybe `AuxPointerType`?